src/views/MarketsDropdown.tsx

```diff
@@ -361,6 +364,14 @@ Styled.Table = styled(Table)`
     }
   }
 
+  tfoot {
+    --stickyArea-totalInsetTop: 0px;
+    --stickyArea-totalInsetBottom: 3px; //xcxc
```

Comment:
ok I know this has to be 3px for possible rendering, but unsure what variables I'm supposed to be setting to get this properly calculated 🫠 would love to pair on this one because everything else looks fine